code/modules/mob/living/simple_animal/bot/griefsky.dm

```diff
@@ -207,6 +207,26 @@
 
 //this section is blocking attack
 
+///Disassembling the bot in a civilized manner with a multitool
+/mob/living/simple_animal/bot/secbot/griefsky/disassemble()
+	walk_to(src,0)
+	var/turf/Tsec = get_turf(src)
+	new /obj/item/assembly/prox_sensor(Tsec)
+	var/obj/item/secbot_assembly/Sa = new /obj/item/secbot_assembly(Tsec)
+	Sa.build_step = 1
+	Sa.overlays += "hs_hole"
+	Sa.created_name = name
+	new /obj/item/robot_parts/r_arm(Tsec)
+	new /obj/item/robot_parts/l_arm(Tsec)
+	if(weapon == /obj/item/melee/energy/sword/saber/)
+		log_and_message_admins("[key_name(usr)] has dismantled [src] containing energy sword(s)]!")
+	new weapon(Tsec)
+	new weapon(Tsec)
+	new weapon(Tsec)
+	new weapon(Tsec)
```

Comment:
Correct me if I'm wrong, but can't you mix and match different types of eswords? Will this allow for esword duping?

Reply:
You can mix and match toy swords and energy swords but the assembly will only be finished and spawn a griefsky if there are four matching types of swords wich will determine the type of the griefsky.
Exemple : I added 2 toy and 2 energy swords to an assembly, then added 2 more toy swords for a total of 4 toy swords and 2 energy swords > The assembly finished and spawned a toy griefsky that once dismantled dropped 4 toy swords.

Reply:
So no esword duping possible. You can however delete eswords if you want!

Reply:
Sounds good!